app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @contestants.each do |contestant| %>
+  <div id="contestant-<%= contestant.id %>">
+    <h3><%= contestant.name %></h3>
+    <p><%= contestant.created_at %></p>
+
+    <% contestant.projects.each do |project| %>
+      <p>Projects: <%= project.name %>
+    <% end %>
```

Comment:
I read in your reflection not wanting to waste any time trying to get the projects to print in a comma-separated list. Here is what a potential solution could have looked like:

```erb
<p>Projects: <%= contestant.projects.map do |project| %>
  <% project.name %>
<% end.join(", ") %></p>
```